P1684-mdarray/P1684.md

```diff
@@ -22,6 +22,13 @@ author:
 #### Changes from R1
 
 - update and reword non-wording text to harmonize with P0009R16
+- fix synopsis missing Alloc argument in some places
+- fix constraints on some constructors to allow std::array as container
+- add missing wording for std::initializer list
+- add constructors from ranges
+- always use std::vector as the default container type, based on LWEG guidance
+- remove container accessor function
+  - its a bad idea to allow someone to resize the container owned by an mdarray ...
```

Comment:
```suggestion
  - it's a bad idea to allow someone to resize the container owned by an `mdarray` ...
```

Reply:
Essentially giving the user a chance to break the class invariance